api/environments/identities/models.py

```diff
@@ -20,6 +23,16 @@
     )
 
 
+def is_dynmodb_configured(f):
+    def inner(*args, **kwargs):
+        if not dynamo_identity_table:
+            return
+        else:
```

Comment:
Being picky now but this `else` is unnecessary. It's neater / more pythonic to write: 

```python
if not dynamo_identity_table:
    raise SomeCustomException()
return f(*args, **kwargs)
```